test/e2e/cypress/e2e/checks_catalog.cy.js

```diff
@@ -44,6 +44,7 @@ context('Checks catalog', () => {
     it('should show 3 check groups in the catalog', () => {
       cy.get('div.check-group').should('have.length', 3);
     });
+
```

Comment:
Adding space between blocks as required from the `mocha/consistent-spacing-between-blocks`. Please not that this rule can't be enforced automatically via prettier (or at least, I wasn't able to do it).